WiFlyHQ.h

```diff
@@ -312,23 +312,23 @@ class WiFly : public Stream {
     boolean setFtpMode(uint8_t mode);
 
     boolean ftpGet(
-	const char *addr,
-	const char *dir,
-	const char *user,
-	const char *password,
-	const char *filename);
+    const char *addr,
```

Comment:
the boolean line had 4 spaces, the lines under it a single tab.  maybe it appeared indented with tab=8 spaces, but i use 4 spaces, so it did not appear indented.  I did nothing more than replace tabs with 4 spaces since that's what the code seemed to use the most.